server/cmwell-dc/src/pack/logback.xml

```diff
@@ -190,7 +214,6 @@
 
     <appender name="ASYNC_TOOLS_RED_LOG" class="ch.qos.logback.classic.AsyncAppender">
         <appender-ref ref="TOOLS_RED_LOG" />
-        <maxFlushTime>0</maxFlushTime>
```

Comment:
same argument here. Why remove the maxFlushTime?

Reply:
Agree. Donw